examples/v1/pipelineruns/no-ci/pipelinerun-with-pipelines-in-pipelines.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: tekton.dev/v1
```

Comment:
Why did we include the example if it's still not there?

Reply:
Thanks @khrm, its a placeholder, we will move it under `alpha` tests once pipelines in pipelines is implemented.

Reply:
I tend to agree that we don't add examples until it is fully supported since I found it could be a bit confusing to users 🤔

Reply:
If PRs merge isn' spread across the release,  it's fine. Otherwise, it's confusing for the user.

Reply:
`no-ci` holds the examples which are not running as part of our CI for various reasons, one of them is all the resources to run that example is not available as part of the CI e.g. https://github.com/tektoncd/pipeline/blob/main/examples/v1/pipelineruns/no-ci/csi-workspace.yaml But the feature is supported if anyone would like to reference it.

The `examples` are not part of the release but If you think its confusing, I am happy to delete it for now.